.gitignore

```diff
@@ -1,7 +1,7 @@
 Pods/
 BuildOutput/
 # VIM temp files
-*.sw*
+*.swp*
```

Comment:
I just found, in earlier [commits](https://github.com/yonadev/yona-app-ios/commit/3bc9a05c77c21b238da84b58d1dcfd1037afc455#diff-a084b794bc0759e7a6b77810e01874f2), you added VIM files to .gitignore with format `*.sw*`. Due to that .swift files are not getting tracked by Git. 
As of now I've modified to `*.swp*`. Please let us know if there is better approach.

Reply:
Even better would be to simply make it *.swp and *.swo